agasc/scripts/mag_estimate_report.py

```diff
@@ -19,7 +20,9 @@ def parser():
                                        ' CxoTime-compatible time stamp.')
     parse.add_argument('--stop', help='Time to stop processing new observations.'
                                       ' CxoTime-compatible time stamp.')
-    parse.add_argument('--out-dir', default=f'./mag_stats_report',
+    parse.add_argument('--input-dir', default='$SKA/data/agasc',
+                       help='Directory containing mag-stats files')
```

Comment:
It's generally good to include the default values in the `help` string. (It seems like `argparse` should be able to do that automatically, but I don't recall this functionality).

Reply:
Good point. I will do that. For some reason, argparse does not do it. The sphinx-argparse module does, so it shows in web docs, but not in the command-line help.